codec_specs.md

```diff
@@ -58,96 +58,159 @@ A timestamp, expressed in [@!RFC3339] that notes when support for the `Codec Map
-* Bytes 2..n: lengths of the first '`#p`' packets, coded in [Xiph-style lacing]({{site.baseurl}}/index.html#lacing). The length of the last packet is the length of the CodecPrivate block minus the lengths coded in these bytes minus one. 
+## Video Codec Mappings
+
+### V_MS/VFW/FOURCC
+
+Codec ID: `V_MS/VFW/FOURCC`
+
+Codec Name: Microsoft (TM) Video Codec Manager (VCM)
+
+Description: The private data contains the VCM structure BITMAPINFOHEADER including the extra private bytes, as [defined by Microsoft](http://msdn.microsoft.com/library/default.asp?url=/library/en-us/gdi/bitmaps_1rw2.asp). The data are stored in little endian format (like on IA32 machines). Where is the Huffman table stored in HuffYUV, not AVISTREAMINFO ??? And the FourCC, not in AVISTREAMINFO.fccHandler ???
+
+Initialisation: `Private Data` contains the VCM structure BITMAPINFOHEADER including the extra private bytes, as defined by Microsoft in https://msdn.microsoft.com/en-us/library/windows/desktop/dd183376(v=vs.85).aspx.
+
+Citation: https://msdn.microsoft.com/en-us/library/windows/desktop/dd183376(v=vs.85).aspx
+
+### V_UNCOMPRESSED
+
+Codec ID: V_UNCOMPRESSED
+
+Codec Name: Video, raw uncompressed video frames
+
+Description: All details about the used colour specs and bit depth are to be put/read from the KaxCodecColourSpace elements.
+
+Initialisation: none
+
+### V_MPEG4/ISO/SP
+
+Codec ID: V_MPEG4/ISO/SP
+
+Codec Name: MPEG4 ISO simple profile (DivX4)
+
+Description: Stream was created via improved codec API (UCI) or even transmuxed from AVI (no b-frames in Simple Profile), frame order is coding order.
+
+Initialisation: none
+
+### V_MPEG4/ISO/ASP
+
+Codec ID: V_MPEG4/ISO/ASP
+
+Codec Name: MPEG4 ISO advanced simple profile (DivX5, XviD, FFMPEG)
+
+Description: Stream was created via improved codec API (UCI) or transmuxed from MP4, not simply transmuxed from AVI! Note there are differences how b-frames are handled in these native streams, when being compared to a VfW created stream, as here there are `no` dummy frames inserted, the frame order is exactly the same as the coding order, same as in MP4 streams!
+
+Initialisation: none
+
+### V_MPEG4/ISO/AP
+
+Codec ID: V_MPEG4/ISO/AP
+
+Codec Name: MPEG4 ISO advanced profile
+
+Description: Stream was created via improved codec API (UCI) or transmuxed from MP4, not simply transmuxed from AVI! Note there are differences how b-frames are handled in these native streams, when being compared to a VfW created stream, as here there are `no` dummy frames inserted, the frame order is exactly the same as the coding order, same as in MP4 streams!
+
+Initialisation: none
+
+### V_MPEG4/MS/V3
+
+Codec ID: V_MPEG4/MS/V3
+
+Codec Name: Microsoft (TM) MPEG4 V3
+
+Description: and derivates, means DivX3, Angelpotion, SMR, etc.; stream was created using VfW codec or transmuxed from AVI; note that V1/V2 are covered in VfW compatibility mode
```

Comment:
I updated the description as suggested to:
> Microsoft (TM) MPEG4 V3 and derivates, means DivX3, Angelpotion, SMR, etc.; stream was created using VfW codec or transmuxed from AVI; note that V1/V2 are covered in VfW compatibility mode.

but I'm uncertain about if that is the intent. The specs page at https://www.matroska.org/technical/specs/codecid/index.html also uses the sentence fragment description. Pinging @robUx4.